src/types.ts

```diff
@@ -1,6 +1,46 @@
+import {
+  EditionMetadataOrUri,
+  NFTMetadataOrUri,
+} from "@thirdweb-dev/sdk/dist/src/schema";
+
 /**
  * Makes a parameter required to be passed, but still allowes it to be undefined.
  *
  * @beta
  */
 export type RequiredParam<T> = T | undefined;
+
+/**
+ * A wallet address.
+ * @beta
+ */
+export type WalletAddress = string;
+
+/**
+ * The parameters to pass to the nft mint function.
+ *
+ * @beta
```

Comment:
do we need @beta ?

Reply:
we don't need it, no

Reply:
but I kinda like having it for the moment because I may still want to change it more